DEVELOPMENT.md

```diff
@@ -32,28 +32,27 @@ This `ota-test_base` image contains a copy of pre-build minimum `ota-image` unde
 docker compose -f docker/test_base/docker-compose_tests.yml run --rm tester
 ```
 
-### Run specific tests manually by override the command
-
-Directly execute pytest is also possible by override the command:
-
-```bash
-docker compose -f docker/test_base/docker-compose_tests.yml run --rm tester \
-   python3 -m pytest /ota-client/tests/<specific_test_file> [<test_file_2> [...]]
-```
-
```

Comment:
This doesn't work, thus suggest to remove.

Reply:
Ah, I didn't update the manual here.

The new way to run specific test files are changed to:
```shell
docker compose -f docker/test_base/docker-compose_tests.yml run --rm tester-ubuntu-20.04 [<test_file_2> [...]]
```

Reply:
That's good to know! thank you. Updated.

Reply:
Notice that there is no `tester` dockercompose service now as we have three testers for Ubuntu 18.04, 20.04 and 22.04.
`tester-ubuntu-20.04` for testing with Ubuntu 20.04 test docker image.

Reply:
OK, added the explanation of `tester` in the latest commit.